internal/pkg/application/transform/transformers_test.go

```diff
@@ -196,33 +217,73 @@ func TestThatWaterConsumptionObservedIsCreatedIfNonExisting(t *testing.T) {
 	is.Equal(string(b), expectedCreateBody)
 }
 
-func TestDeltaVolumes(t *testing.T) {
-	is := is.New(t)
+// GreenspaceRecord test notes:
+// Pressure and Condctivity may come as array of values from iot-core.
+//  - first occurances of these are treated as primary measurement
+//	- subsequent occuranceses are treated as additional measurements and ignored
```

Comment:
sp. of occurance on line 222 - should be singular. "first occurance is treated.."

on line 223, should be occurances.